Credify/Credify/Objects/Entities.swift

```diff
@@ -273,3 +335,235 @@ public enum ProductType : String {
 //    case consumerBNPL = "consumer"
 //    case merchantBNPL = "merchant"
 }
+
+/// Reference on this site https://app.quicktype.io/#l=swift
+public class AnyValue: Codable {
```

Comment:
Please separate files. This file should focus on Credify related objects only.

Reply:
Moved it to `CommonModel` file